pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtNDManager.java

```diff
@@ -118,13 +118,24 @@ public PtNDArray create(Buffer data, Shape shape, DataType dataType) {
     /** {@inheritDoc} */
     @Override
     public NDArray createCSR(Buffer data, long[] indptr, long[] indices, Shape shape) {
-        throw new UnsupportedOperationException("Not implemented");
+        throw new UnsupportedOperationException("PyTorch doesn't support CSR");
     }
 
     /** {@inheritDoc} */
     @Override
     public NDArray createRowSparse(Buffer data, Shape dataShape, long[] indices, Shape shape) {
-        throw new UnsupportedOperationException("Not implemented");
+        throw new UnsupportedOperationException("PyTorch doesn't support Row Sparse");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public NDArray createCOO(Buffer data, long[] indices, Shape indicesShape, Shape shape) {
+        // length should be the same as indices dim 1
+        try (NDArray valueNd = create(data, new Shape(indicesShape.get(1)))) {
```

Comment:
```suggestion
        try (NDArray valueNd = create(data, new Shape(indicesShape.get(indicesShape.dimension() - 1)))) {
```
1 only applied for 2-D

Reply:
In 3D, the indices shape will be in (3, num_of_elements)